src/frontend/apps/standalone_site/src/features/Profile/features/AccountSettings/components/AccountSettings.tsx

```diff
@@ -32,45 +32,45 @@ const messages = defineMessages({
     description: 'Label for current password text input.',
     id: 'features.Profile.AccountSettings.currentPassword',
   },
-  showCurrentPassword: {
-    defaultMessage: 'Show the current password.',
-    description: 'A18n label for the show current password button.',
-    id: 'features.Profile.AccountSettings.showCurrentPassword',
+  revealCurrentPassword: {
+    defaultMessage: 'Reveal the current password.',
+    description: 'A18n label for the revealed current password button.',
+    id: 'features.Profile.AccountSettings.revealCurrentPassword',
   },
-  hiddeCurrentPassword: {
-    defaultMessage: 'Hidde the current password.',
-    description: 'A18n label for the hidden current password button.',
-    id: 'features.Profile.AccountSettings.hiddeCurrentPassword',
+  obscureCurrentPassword: {
+    defaultMessage: 'Hide the current password.',
```

Comment:
```suggestion
    defaultMessage: 'Obscure the current password.',
```

Reply:
First time I see this word in this context. I always saw `Hide` and `reveal`.

Reply:
https://github.com/openfun/marsha/pull/2374/commits/90d8ceefdca8d6e811f7616c2fda464415dfe223